### DISCUSSION TITLE
Custom Sample Rate of 44,1 k

### DISCUSSION THREAD

**Discussion by devidw** [yl4579/StyleTTS2#135] [Q&A] [answered]
We have a dataset of wav files with a sample rate of 44,100 that we would like to train from scratch

We noticed the configuration option in the training for `preprocess_params.sr` with a default of 24k

It looks like the ASR module was trained with 24k as well

https://github.com/yl4579/StyleTTS2/blob/d05a46356ee9f85050a843c1f257cd53c9b08dc6/Utils/ASR/config.yml#L14

Wondering if it's fine, to just have the training dataset at 44,1k or if we would need to also have the ASR component being trained at 44,1k for the overall architecture to work?

Thx

**Comment by martinambrus** [comment, marked as answer]
From README:

> The text aligner and pitch extractor are pre-trained on 24 kHz data, but you can easily change the preprocessing and re-train them using your own preprocessing.

**Comment by yl4579** [comment]
You don't even need to re-train. You can just downsample it to 24 kHz and get the melspectrograms for text aligner. Of course the best way is to re-train because you don't have to align the 24 kHz mel to 44.1 kHz waveform.
